### ISSUE TITLE
Device.os returns undefined on ipad

### ISSUE THREAD

**Issue by eradin** [framework7io/framework7#3622]
* Framework7 version: [ 5.7.0, perhaps earlier versions too]
* Platform and Target: [ iOS 13 Cordova app

### Describe the bug
Running app on ipad ios 13.3.1 and calling app.device.os.  It returns undefined.  Works correctly on android and iphone.  I hadn't noticed this before but I did recently add WKWebVIew plugin to be compliant with the app store policies.  Not sure if that is an issue.

### To Reproduce
Just instantiate the framework on an ipad and call app.device.os.

### Expected behavior
should return ios

### Actual Behavior
undefined

### Screenshots
None available but I've confirmed via console,log.

**Comment by nolimits4web** [comment]
Can't replicate it. Would be good to see more details, what is in `device` data? Screen width/height, etc.

**Comment by eradin** [comment]
It's an ipad (6th gen) screen: 9.7" Color*     Display Resolution:     2048x1536 (264 ppi)

Device values from console.log:

{"ios":false,"android":false,"androidChrome":false,"desktop":true,"iphone":false,"ipod":false,"ipad":false,"edge":false,"ie":false,"firefox":false,"macos":false,"windows":false,"cordova":true,"phonegap":true,"electron":false,"nwjs":false,"webView":false,"webview":false,"standalone":false,"pixelRatio":2}

Notice desktop =true and there is no os property.
I have rebooted the device with the same results.  Not that it shouldn't matter but I built this using the ios13 sdk and the WkWebView plugin is cordova-plugin-ionic-webview" spec="4.2.0".

Confirmed iphone 11 running 13.4.1 works and device settings all look good.
